wicket-native-websocket/wicket-native-websocket-core/src/main/java/org/apache/wicket/protocol/ws/api/WebSocketPushBroadcaster.java

```diff
@@ -189,7 +189,20 @@ private void process(final Application application, final Collection<IWebSocketC
 				@Override
 				public void run()
 				{
-					wsConnection.sendMessage(message);
+					if (webSocketSettings.isAsynchronousPush())
+					{
+						if (webSocketSettings.getAsynchronousPushTimeout() > 0)
```

Comment:
I would drop this `if` and call the function with `timeout`

Reply:
Done